Formula/b/bigquery-emulator.rb

```diff
@@ -0,0 +1,35 @@
+class BigqueryEmulator < Formula
+  desc "Emulate a GCP BigQuery server on your local machine"
+  homepage "https://github.com/goccy/bigquery-emulator"
+  url "https://github.com/goccy/bigquery-emulator/archive/refs/tags/v0.6.1.tar.gz"
+  sha256 "4f8c037d03cd23d2a44d74460b5399213e0efeb33d6cade25bfce25499c4699a"
+  license "MIT"
+  head "https://github.com/goccy/bigquery-emulator.git", branch: "main"
+
+  depends_on "go" => :build
+  depends_on "llvm" => :build
+
+  def install
+    ENV["CGO_ENABLED"] = "1"
+    ENV.clang
+
+    ldflags = "-s -w -X main.version=#{version}"
+    system "go", "build", *std_go_args(ldflags:), "./cmd/bigquery-emulator"
+  end
+
+  test do
+    output = shell_output("#{bin}/bigquery-emulator 2>&1", 1)
+    assert_match "the required flag --project was not specified", output
+
+    assert_match "version: #{version} ()", shell_output("#{bin}/bigquery-emulator --version")
+
+    listen_port = free_port
+
+    fork do
+      exec bin/"bigquery-emulator", "--project=test", "--port=#{listen_port}"
+    end
+
+    sleep 1
+    system "nc", "-z", "localhost", listen_port
```

Comment:
we can just drop the `--project` flag test

```suggestion
    port = free_port

    fork do
      exec bin/"bigquery-emulator", "--project=test", "--port=#{port}"
    end

    sleep 1
    system "nc", "-z", "localhost", port.to_s

    assert_match "version: #{version} ()", shell_output("#{bin}/bigquery-emulator --version")
```